services/user.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth import get_user_model
+from django.shortcuts import get_object_or_404
+
+import init_django_orm  # noqa: F401
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> None:
+    get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email if email else None,
+        first_name=first_name if first_name else "",
+        last_name=last_name if last_name else ""
+    )
+
+
+def get_user(user_id: int) -> User:
```

Comment:
Ensure that the `User` model imported from `db.models` is the same as the one returned by `get_user_model()`. If they differ, it might cause inconsistencies.